### PR TITLE
dotCMS/core#17795 Fix dupe view and add multiselect with shift to card view

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: dotCMS/src/main/webapp/dotcms-webcomponents/p-6qksgvke.system.entry.js [hunk 1]
```diff
@@ -0,0 +1 @@
...(t){var n=e.el.shadowRoot.querySelectorAll("dot-card-contentlet");var r=t.target;var i=false;if(t.shiftKey&&r.checked){n.forEach((function(t){if(t===r||t===e.lastChecked){i=!i}if(i){t.checked=true}}))}e.lastChecked=r},key:n.data.inode,checked:t.includes(n.data.inode),onCheckboxChange:function(t){var n=t.detail,r=n.originalTarget,i=n.data;if(r.checked){e.selection.push(i)}else{e.selection=e.selection.filter((function(e){return e.identifier!==i.identifier}))}e.value=e.selection.map((function(e){var t=e.inode;return t})).join(",");e.selected.emit(e.selection)},item:n})})))};Object.defineProperty(e.prototype,"el",{get:function(){return o(this)},enumerable:true,configurable:true});Object.defineProperty(e,"watchers",{get:function(){return{items:["watchItems"],value:["watchValue"]}},enumerable:true,configurable:true});Object.defineProperty(e,"style",{get:function(){return":host{display:grid;grid-template-columns:repeat(auto-fill,minmax(220px,1fr));grid-gap:var(--basic-padding-2)}dot-card-contentlet{height:100%}dot-card-contentlet:before{content:\"\";display:inline-block;-ms-flex:0 0 0px;flex:0 0 0;height:0;padding-bottom:calc(100%)}"},enumerable:true,configurable:true});return e}())}}}));
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/dotcms-webcomponents/p-6qksgvke.system.entry.js]: Codacy Issue found: [Avoid assignments in operands](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5139982)

Comment (dev-dotcms) [inline dotCMS/src/main/webapp/dotcms-webcomponents/p-6qksgvke.system.entry.js]: Codacy Issue found: [Avoid using global variables](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5139982)

Comment (dev-dotcms) [inline dotCMS/src/main/webapp/dotcms-webcomponents/p-6qksgvke.system.entry.js]: Codacy Issue found: [Avoid using while statements without curly braces](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5139982)

File: dotCMS/src/main/webapp/dotcms-webcomponents/p-xwmq3ozs.entry.js [hunk 2]
```diff
@@ -0,0 +1 @@
...sync getValue(){return this.selection}watchItems(t){this.selection=h(t,this.value)}watchValue(t){this.selection=h(this.items,t)}render(){const t=(t=>t&&"string"==typeof t?t.split(","):[])(this.value);return e(i,null,this.items.map(s=>e("dot-card-contentlet",{onClick:t=>{const s=this.el.shadowRoot.querySelectorAll("dot-card-contentlet"),e=t.target;let i=!1;t.shiftKey&&e.checked&&s.forEach(t=>{t!==e&&t!==this.lastChecked||(i=!i),i&&(t.checked=!0)}),this.lastChecked=e},key:s.data.inode,checked:t.includes(s.data.inode),onCheckboxChange:({detail:{originalTarget:t,data:s}})=>{t.checked?this.selection.push(s):this.selection=this.selection.filter(t=>t.identifier!==s.identifier),this.value=this.selection.map(({inode:t})=>t).join(","),this.selected.emit(this.selection)},item:s})))}get el(){return a(this)}static get watchers(){return{items:["watchItems"],value:["watchValue"]}}static get style(){return":host{display:grid;grid-template-columns:repeat(auto-fill,minmax(220px,1fr));grid-gap:var(--basic-padding-2)}dot-card-contentlet{height:100%}dot-card-contentlet:before{content:\"\";display:inline-block;-ms-flex:0 0 0px;flex:0 0 0;height:0;padding-bottom:calc(100%)}"}};export{c as dot_card_view};
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/dotcms-webcomponents/p-xwmq3ozs.entry.js]: Codacy Issue found: [Avoid using global variables](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5139982)

File: dotCMS/src/main/webapp/dotcms-webcomponents/p-6qksgvke.system.entry.js (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/dotcms-webcomponents/p-6qksgvke.system.entry.js]: Codacy Issue found: [A 'return', 'break', 'continue', or 'throw' statement should be the last in a block.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5139982)

Comment (dev-dotcms) [inline dotCMS/src/main/webapp/dotcms-webcomponents/p-6qksgvke.system.entry.js]: Codacy Issue found: [Avoid using if statements without curly braces](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5139982)

**Review by wezell — APPROVED** [review]

**Pull request merged**